src/uct/ib/base/ib_iface.c

```diff
@@ -176,6 +176,10 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "path for the same pair of endpoints.",
    ucs_offsetof(uct_ib_iface_config_t, roce_path_factor), UCS_CONFIG_TYPE_UINT},
 
+  {"RANDOM_ROCE_PATH_FACTOR", "n",
+   "Enable/Disable roce path factor random generation.",
```

Comment:
"Enable/Disable random RoCE path generation",

Reply:
@yosefe Fixed.

Reply:
looks like not fixed